.releaserc.cjs

```diff
@@ -54,6 +54,23 @@ const Runtimes = {
       assemblyInfo: "StandardLibrary/runtimes/net/AssemblyInfo.cs",
     },
   },
+  python: {
+    "AwsCryptographicMaterialProviders/runtimes/python/pyproject.toml": {
```

Comment:
nit: maybe leave a comment stating that dependency changes here will need to be accounted for in `CheckDependencyReplacementResults`